VSInsertion/Signing/SignFiles.csproj

```diff
@@ -46,7 +46,6 @@
     <ItemGroup>
       <FilesToSign Include="@(FilesToCopy->'$(OutDir)%(Filename)%(Extension)')">
         <Authenticode>Microsoft</Authenticode>
```

Comment:
Why did you remove this?

Reply:
Because I'm experimenting. Looking at the logs of the build definition I'm writing and the build definition of the existing VCLS signing that we manually trigger, they both say strong name authentication cannot be done (there's a warning) but then the exes get submitted to some server for signing. I'm trying to remove things we don't need to reduce confusion in the future and want to see if this removes that warning.